contracts/governance/extensions/GovernorStorage.sol

```diff
@@ -100,14 +100,19 @@ abstract contract GovernorStorage is Governor {
      */
     function proposalDetailsAt(
         uint256 index
-    ) public view virtual returns (uint256, address[] memory, uint256[] memory, bytes[] memory, bytes32) {
-        uint256 proposalId = _proposalIds[index];
-        (
+    )
+        public
+        view
+        virtual
+        returns (
+            uint256 proposalId,
             address[] memory targets,
             uint256[] memory values,
             bytes[] memory calldatas,
             bytes32 descriptionHash
-        ) = proposalDetails(proposalId);
-        return (proposalId, targets, values, calldatas, descriptionHash);
+        )
+    {
+        proposalId = _proposalIds[index];
+        (targets, values, calldatas, descriptionHash) = proposalDetails(proposalId);
```

Comment:
The guidelines are about putting names on the return values, but not necessarily about using those variables. What are our guidelines for actually using those variables instead of explicit return statements?

Reply:
Naming the return variable implies declaring them. This means the name is used and cannot be redeclared. The example here is a good one: do we want to redeclare everything with a different name, and then do an explicit return ? Sometimes maybe, but here it felt too much.